src/app/cart/directives/highlight.directive.ts

```diff
@@ -0,0 +1,22 @@
+import { Directive, ElementRef, HostListener, Renderer2 } from '@angular/core';
+
+@Directive({
+  selector: '[appHighlight]',
+})
+export class HighlightDirective {
+  constructor(private el: ElementRef, private renderer: Renderer2) {}
+  @HostListener('mouseenter')
+  itsJustFunctionsName() {
+    this.highlightOn();
+  }
+  @HostListener('mouseleave')
+  function2() {
```

Comment:
change the name to `onMouseLeave`

Reply:
'mouseleave' it is a standart HTML DOM event https://www.w3schools.com/jsref/dom_obj_event.asp, that is passed to @HostListener https://angular.io/api/core/HostListener#options

Reply:
I don't make you change the name of the event but the name of the method  ` function2`